airbyte-ci/connectors/base_images/base_images/__init__.py

```diff
@@ -0,0 +1,63 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+import sys
+from itertools import product
+from pathlib import Path
+
+import anyio
+import dagger
+from base_images import common, consts, utils
+from rich.console import Console
+
+console = Console()
+
+try:
+    from base_images import python_bases  # , java_bases
+except common.BaseImageVersionError as e:
```

Comment:
This error occurs if a base image version class name does not follow semver. We handle the error for nice console output. It might happen if a developer implement a new version class without following our required class name convention.